README.md

```diff
@@ -38,34 +30,31 @@ To use a snapshot, add the following repository to your `pom.xml`:
 
 ## Usage
 
-The provider is instantiated using a client token that is configured in the Confidence UI or via the
-management API. After that all interaction with the feature flags happens using the OpenFeature client APIs. 
+The SDK is instantiated using a client token that is configured in the Confidence UI or via the
+management.
 
+### Resolving flags
+Flag values are evaluated remotely and returned to the application:
 ```java
-package com.spotify.confidence.openfeature;
-
-import com.spotify.confidence.ConfidenceFeatureProvider;
-import dev.openfeature.sdk.Client;
-import dev.openfeature.sdk.MutableContext;
-import dev.openfeature.sdk.OpenFeatureAPI;
-import dev.openfeature.sdk.Value;
-import java.util.Map;
-
-public final class ResolveFlags {
+final Confidence confidence = Confidence.builder("<CLIENT_TOKEN>").build();
+confidence.setContext(Map.of("country", ConfidenceValue.of("SE")));
```

Comment:
I would actually advocate for people using something like this:
```

final String propertyValue = confidence.withContext(Map.of(
                "user_id", ConfidenceValue.of("<some-user-id>"), 
                "country", ConfidenceValue.of("SE"))
        ).getValue(
    "flag-name.property-name",
    "defaultValue"
);
```

At least we should not use `country` as an example for mutating the root context.